File: Neps-Academy/fase0 - SBC - 2025/L.cpp
#include <bits/stdc++.h>
using namespace std;
typedef long long ll;

int main() {
  ll m; cin >> m;
  ll total = 8LL * 1000000LL * m;
  int n = 0;
  ll power = 1;
  while (power < total) {
    power *= 2;
    n++;
  }
  cout << n << endl;
}

File: Neps-Academy/fase0 - SBC - 2025/L.go
package main
import (
	"fmt"
)

func main() {
	var n int64;
	fmt.Scanf("%d", &n);
	var m int = 0;
	var power int64 = 1;
	var total int64 = 8 * 1000000 * n;
	for power < total {
		power = power * 2;
		m++;
	}
	fmt.Printf("%d\n", m);
}